commands/cluster-slots.md

```diff
@@ -34,7 +34,8 @@ Each nested result is:
   - ...continues until all replicas for this master are returned.
 
 Each result includes all active replicas of the master instance
-for the listed slot range.  Failed replicas are not returned.
+for the listed slot range. These active replicas are ordered deterministically.  
```

Comment:
I think we could just mention that the command response is deterministic across all nodes in a cluster. WDYT ?

Reply:
Also, should we clarify that they are ordered lexographically?

Reply:
I think we should. Because  clients could check the result according to their ipaddress (becuase ipaddress as the first item in the result).

Reply:
>I think we could just mention that the command response is deterministic across all nodes in a cluster

Do you mean at the start of the doc? like this
```
`CLUSTER SLOTS` returns details about which cluster slots map to which Valkey instances. The command response is deterministic across all nodes in a cluster and is ordered lexicographically.
```

Reply:
I'm fine with the placement at the top.